mmv1/third_party/terraform/services/vmwareengine/resource_vmwareengine_cluster_sweeper.go

```diff
@@ -57,16 +55,21 @@ func testSweepVmwareengineCluster(region string) error {
 		}
 
 		log.Printf("[INFO][SWEEPER_LOG] looking for parent resources in location '%s'.", location)
-		privateCloudNames, err := listPrivateCloudsInLocation(d, config)
+
+		parentResponseField := "privateClouds"
+		parentListUrlTemplate := "https://vmwareengine.googleapis.com/v1/projects/{{project}}/locations/{{location}}/privateClouds"
+		parentNames, err := sweeper.ListParentResourcesInLocation(d, config, parentListUrlTemplate, parentResponseField)
 		if err != nil {
 			log.Printf("[INFO][SWEEPER_LOG] error finding parental resources in location %s: %s", location, err)
 			continue
 		}
-		for _, parent := range privateCloudNames {
+		for _, parent := range parentNames {
 
 			// `parent` will be string of form projects/my-project/locations/us-central1-a/privateClouds/my-cloud
-			listUrl := fmt.Sprintf("https://vmwareengine.googleapis.com/v1/projects/%s/clusters", parent)
```

Comment:
Also fixed a defect in this url where `/projects/projects` would be in the final templated URL